src/expr/datatype.h

```diff
@@ -307,6 +333,13 @@ class CVC4_PUBLIC DatatypeConstructor {
   unsigned getNumSygusLetInputArgs() const;
   /** is this a sygus identity function */
   bool isSygusIdFunc() const;
+  /** get sygus print callback
+   * This class stores custom ways of printing
```

Comment:
The documentation is not consistent. Sometimes there is a new line after a brief, sometimes no brief and so on.